new-sequence-alerts-config.json

```diff
@@ -0,0 +1,8 @@
+[{
```

Comment:
I think my only comment would just be - if you don't have any entries for `region` etc, can they be left out, or must be there but empty? This would be good to maybe specify explicitly so we don't end up breaking it by removing 'empty' lines... (or make it so it can be removed?)

Reply:
Excellent point. I'm going to slightly modify the logic of how we loop over each entry in the config file, and when I do so, I'll make sure that empty `region` etc. can be omitted without throwing an error.